west.yml

```diff
@@ -21,6 +21,10 @@ manifest:
   remotes:
     - name: upstream
       url-base: https://github.com/zephyrproject-rtos
+    - name: apache
+      url-base: https://github.com/apache
+    - name: juul
+      url-base: https://github.com/JuulLabs-OSS
```

Comment:
Is it intentional to change the remote?

I thought we were using forks intentionally.

This change needs to be elaborated on in the commit message.

Reply:
Yes, but process is that first upstreams are changed, then these changes are picked up to our forks. SO basically I have to go through review there and then move changes to our forks.

Reply:
Okay, so this PR will not be merged until after we have removed this patch to west.yml, right?

Reply:
Yes, this part will be removed. This patch is basically provided to update west and gather all changes together for tests.

Reply:
@de-nordic patch available in https://github.com/zephyrproject-rtos/mcuboot now

Reply:
Thanks!